### DISCUSSION TITLE
Widget background color

### DISCUSSION THREAD

**Discussion by jblindsay** [emilk/egui#2267] [Q&A] [answered]
Hello,
Where can I find the 'blue' colour used for interactive widgets like the progress bar and a toggle value? I've looked in visuals, and while many other of the theme colours are provided there, it does not appear that this particular colour is. I've been using the hyperlink colour, simply because it's close, but it's not the same. Thanks.

**Comment by jblindsay** [comment, marked as answer]
I've answered my own question....it's visuals().selection.bg_fill. ;-)

**Reply by coderedart** [reply]
mark as answered :)
